### PR TITLE
blog post for Cordova node xcode 1.1.0 release

### PR DESCRIPTION
**Pull request by brodybits**

### CODE REVIEW

**Comment by janpio** [comment]
Ups, misclicked.

Does this really need a blog post? It is only slightly related to Cordova, not targeted at actual Cordova end users, and also doesn't really change anything.

File: www/_posts/2018-12-21-xcode-1.1.0.md [hunk 1]
```diff
@@ -0,0 +1,25 @@
+---
+layout: post
+author:
+    name: Chris Brody
+    url: https://twitter.com/brodybits
+title:  "Cordova node xcode 8.1.2 Patch Release"
```
Comment (erisu) [inline www/_posts/2018-12-21-xcode-1.1.0.md]: `8.1.2`??? You mean `1.1.0`???

Comment (brodybits) [reply www/_posts/2018-12-21-xcode-1.1.0.md]: Should be fixed now

**Comment by janpio** [comment]
So you just merge the PR instead of answering my question?

**Comment by brodybits** [comment]
> Does this really need a blog post?

I think we should add it since it can slightly change behavior on cordova-ios and cordova-osx scripts.

I think it would be ideal to move cordova-node-xcode out of the Cordova project, just raised <https://github.com/apache/cordova-node-xcode/issues/42>.

Should we consider reverting this one?

**Pull request merged**